src/assets/questionsAndAnswers/questionsAndAnswers.js

```diff
@@ -175,7 +175,7 @@ export default {
                     {
                         "id": "useQuestion1",
                         "question": "What does this map show me?",
-                        "answer": "The maps show seasonal-average daily average water withdrawals during 2015 divided into 3 month periods for thermoelectric, irrigation, and public supply categories of use within each watershed (<a\n" +
```

Comment:
We should remove one "average" from this phrase.  
"the maps show seasonal-average daily water withdrawals...."  seems to work for me.

Reply:
I like the red better, which I believe is the old text.

Reply:
ok, i put the original text back with your suggested edit! thanks!